azkaban-common/src/main/c/execute-as-user.c

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
Is this being built with the build system? I don't see any build.gradle changes here.